### PR TITLE
Allow setting custom template for crep report output

### PR DESCRIPTION
**Pull request by jamuc**
Allow specifying your template for the report generation

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Next up
+- Allow supplying custom templates for the crash report via command line argument
```
Comment (BasThomas) [inline Changelog.md]: Can you put this under "added"?

File: Readme.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ Run Crep:
 
 The `--only-unresolved` flag filters out any crashes marked as resolved.
 
+ERB templates are used for the crash report layout. You can either use the supplied template in the templates folder
+or specify your own via command line argument.
```
Comment (BasThomas) [inline Readme.md]: Nit: `via a command line argument.`

File: Readme.md [hunk 3]
```diff
@@ -26,6 +26,11 @@ Run Crep:
...
+Example: `bundle exec crep crashes --identifier .... --template=<path to your template>```
```
Comment (BasThomas) [inline Readme.md]: I think you meant to only use one backtick at the end? :)

File: Readme.md [hunk 4]
```diff
@@ -26,6 +26,11 @@ Run Crep:
...
+For more information on [ERB](http://www.stuartellis.name/articles/erb/)
```
Comment (BasThomas) [inline Readme.md]: What about giving an example / link to the current template? You could add a link to the `ERB` [on line 29](https://github.com/xing/crep/pull/75/files#diff-1e290ac8433d555bce009b162cb869d0R29) and omit this.

File: lib/crep/command/crashes.rb [hunk 5]
```diff
@@ -24,17 +26,18 @@ def self.options
...
-      Besides the optional `top`, `verbose` and `only-unresolved` parameters, Crep requires all other parameters.
+      Besides the optional `top`, `template` and `only-unresolved` parameters, Crep requires all other parameters.
     DESC
 
     def initialize(argv)
       super
 
       @show_only_unresolved = argv.flag?('only-unresolved', false)
-      @top = argv.option('top') || DEFAULT_TOP_COUNT
+      @top = argv.option('top', DEFAULT_TOP_COUNT)
```
Comment (BasThomas) [inline lib/crep/command/crashes.rb]: Nice!

File: lib/crep/command/crashes.rb [hunk 6]
```diff
@@ -24,17 +26,18 @@ def self.options
     DESC
 
     self.description = <<-DESC
-      Besides the optional `top`, `verbose` and `only-unresolved` parameters, Crep requires all other parameters.
+      Besides the optional `top`, `template` and `only-unresolved` parameters, Crep requires all other parameters.
```
Comment (BasThomas) [inline lib/crep/command/crashes.rb]: 👍

File: spec/crash_controller_spec.rb [hunk 7]
```diff
@@ -4,7 +4,8 @@
 
 
 RSpec.describe Crep::CrashController do
-  subject { described_class.new(app.bundle_identifier, 5, crash_source, true) }
+  TEMPLATE = 'templates/default_crash_report.txt.erb'
+  subject { described_class.new(app.bundle_identifier, 5, crash_source, true, TEMPLATE) }
```
Comment (BasThomas) [inline spec/crash_controller_spec.rb]: Lovely 😊

**Comment by q231950** [comment]
Thanks for the implementation @jamuc + the review @BasThomas 👍

**Pull request merged**